tests/conftest.py

```diff
@@ -7,4 +7,18 @@
     https://pytest.org/latest/plugins.html
 """
 
-# import pytest
+import pytest
+import syft as sy
+
+
+@pytest.fixture
+def clients():
+    alice = sy.VirtualMachine(name="alice")
+    bob = sy.VirtualMachine(name="bob")
+    james = sy.VirtualMachine(name="james")
+
+    alice_client = alice.get_client()
+    bob_client = bob.get_client()
+    james_client = james.get_client()
+
+    return [alice_client, bob_client, james_client]
```

Comment:
Q: Could you use only 2 workers for the time being? If we use >2 workers there is a chance we get an error at the multiplication because of wraparound.